eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/log/EventLog.scala

```diff
@@ -605,6 +605,19 @@ abstract class EventLog[A <: EventLogState](id: String) extends Actor with Event
 }
 
 object EventLog {
+
+  /**
+   * Published on the event-stream, when the [[CircuitBreaker]] closes as a write finally
+   * was successful (after n retries).
+   */
+  case class EventLogAvailable(logId: String)
```

Comment:
Why don't you reuse `ServiceEvent`s and publish them directly on the event stream? They'd need an additional `logId` field.